foxy/distribution.yaml

```diff
@@ -2836,11 +2836,10 @@ repositories:
       - ros2_control
       - ros2controlcli
       - test_robot_hardware
-      - transmission_interface
```

Comment:
@bmagyar I don't know if you've seen my comment (https://github.com/ros/rosdistro/pull/27828#issuecomment-750405485), but I think the build error should be easy to fix by adding a test dependency to the package.xml.

Reply:
I did and thanks for that. Still had another issue to fix and transmission_interface is not a first release priority